packages/admin/admin-theme/src/componentsTheme/MuiDataGrid.ts

```diff
@@ -0,0 +1,71 @@
+import type {} from "@mui/x-data-grid/themeAugmentation";
+
+import { mergeOverrideStyles } from "../utils/mergeOverrideStyles";
+import { GetMuiComponentTheme } from "./getComponentsTheme";
+
+export const getMuiDataGrid: GetMuiComponentTheme<"MuiDataGrid"> = (component) => ({
+    ...component,
+    styleOverrides: mergeOverrideStyles<"MuiDataGrid">(component?.styleOverrides, {
+        root: {
+            backgroundColor: "white",
+        },
+        columnHeader: {
+            "&:focus": {
+                outline: "none",
+            },
+            "&:focus-within": {
+                outline: "none",
+            },
+        },
+        columnHeadersInner: {
+            padding: "0 10px",
+        },
+        row: {
+            padding: "0 10px",
+        },
+        cell: {
+            borderTop: "1px solid #D9D9D9",
```

Comment:
You can use the theme values here: 

```suggestion
            borderTop: `1px solid ${palette.grey[100]}`,
```